addons/binding/org.openhab.binding.regoheatpump/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Added the new line, however not sure where/how to properly add the about.html? I briefly checked other bindings and didn't find one that would include the about.html file.